stacks/atm0s-media-stack/deploy.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -e
+
+################################################################################
+# repo
+################################################################################
+helm repo add 8xff https://8xff.github.io/helm/
+helm repo update > /dev/null
+
+################################################################################
+# chart
+################################################################################
+STACK="atm0s-media-stack"
+CHART="8xff/atm0s-media-stack"
+CHART_VERSION="0.1.2"
```

Comment:
Helm chart version 0.1.2 is not available. Only version I see is 0.1.0
![Screenshot 2024-01-24 at 8 22 58 PM](https://github.com/digitalocean/marketplace-kubernetes/assets/36917391/18c04e23-65bb-4cbf-bfe6-232aad1d1959)